.github/workflows/docker.yml

```diff
@@ -62,10 +62,20 @@ jobs:
       - id: default
         if: github.event_name != 'schedule'
         run: echo "R_VERSION=${{ github.event.inputs.r_version || env.DEFAULT_R_VERSION }}" >> "$GITHUB_OUTPUT"
+      - id: platform
+        # upstream rocker/tidyverse image is only on ARM for R 4.5.* (so far)
+        shell: bash
+        run: |
+          if [[ "${{ join(steps.*.outputs.R_VERSION, '') }}" == "4.5" ]]; then
```

Comment:
more complex, but do we want to use explicit if 4.1 the x86, etc, then else arm. That way develop also compiles with arm. Once we disable 4.2 and add 4.6 it does not need any updates, except removal of 4.2